package/MDAnalysis/analysis/density.py

```diff
@@ -107,7 +107,7 @@
       After the analysis (see the :meth:`~DensityAnalysis.run` method), the resulting density is
       stored in the :attr:`density` attribute as a :class:`Density` instance.
 
-   .. automethod:: _set_user_grid
+   .. automethod:: _prepare, _set_user_grid
```

Comment:
Remove `_prepare`: this should not be documented as it's an internal method. Move the docs into the class doc string. This is the standard way how we document Analysis classes.

`_set_user_grid()` is a special case because it does all kinds of things and one might need to know about it.